uaclient/cli.py

```diff
@@ -495,6 +513,13 @@ def enable_parser(parser):
     parser.add_argument(
         "--beta", action="store_true", help="allow beta service to be enabled"
     )
+    parser.add_argument(
+        "--format",
+        action="store",
+        choices=["cli", "json"],
```

Comment:
I like the idea of changing both `tabular` and `cli` to `plain`.
However, since this change will modify `status` as well, I think we can address this in a separate PR